hell/adventcalendar/2025/11/index.md

```diff
@@ -1,35 +1,119 @@
 ---
 title: "How HTML changes in ePub"
-author: "Your Name"
-author_bio: "Your short bio"
+author: "Robin Whittleton"
+author_bio: "Still calling myself a front-end developer, though these days more focused on spreading the good word of accessibility."
 date: 2025-12-11
 author_links:
   - label: "Site"
-    url: "https://linktoyourblog123.com"
-    link_label: "linktoyourblog123.com"
-intro: "<p>Short introductory text</p>"
+    url: "https://www.robinwhittleton.com/"
+    link_label: "robinwhittleton.com"
+  - label: "Mastodon"
+    url: "https://front-end.social/@robinwhittleton"
+    link_label: "front-end.social/@robinwhittleton"
+intro: "<p>Robin explains why building an ePub might be both easier and harder than you’d think.</p>"
 image: "advent25_11"
 ---
-Some text.
-Some text.
 
-Some text. Some text.
+ePub: the [specification](https://www.w3.org/TR/epub-33/) you always knew you wanted yet never knew how to use. You get to take your knowledge of the web, and use it to produce little self-contained sets of documents that can be freely distributed as a single file ready for reading on extremely low-power devices, and they even reflow to fit any screen.
 
-## Heading
+Yet while I said that you can use your knowledge of the web to build ePubs, the technology in use is twisted in unforeseen ways, and you might have to unlearn the things you think you knew. Prepare yourself…
 
-Some text.
+## HTML, sort of
 
-## Heading
+ePubs, at their core, use HTML, just like the websites we build every day. Except, well, there’s a big asterisk after that. Let’s dive into the differences.
 
-Some text.
+A few decades ago XML emerged from the pit. XML – an attempt at an extensible standard for expressing marked up data – could be used for documents, data transfer, and a bunch of other things, and people genuinely liked it (or, much like AI today, pretended to for job security). They liked it so much that a concerted effort was started to take the nascent HTML and rebuild it on top of XML. This project had a name you might have heard of: XHTML.
```

Comment:
Link XHTML. Either point to the spec or a blog post explaining the difference between HTML4 and XHTML.